docs/GDPs/gdp-0000.rst

```diff
@@ -0,0 +1,269 @@
+:Status: Draft
+:Type: Process
+:Created: 24-03-2020
+:Discussion PR: ...
+
+
+What is a GDP?
+--------------
+
+GDP stands for GT4Py Development Proposal. A GDP is a design document
+providing information to the GT4Py developers community, describing
+a new feature for GT4Py (or GTScript) or changes in the development processes.
+The GDP should provide a concise technical specification and a rationale
+of the new feature, possibly with some examples.
+
+We intend GDPs to be the primary mechanisms for proposing major new
+features, for collecting community input on an issue, and for documenting
+the design decisions that have gone into GT4Py. The GDP author is
+responsible for building consensus within the community and documenting
+dissenting opinions.
+
+Because the GDPs are maintained as text files in a versioned
+repository, their revision history is the historical record of the
+feature proposal, which is available by the normal git commands for
+retrieving older revisions from the ``doc/wiki/GDPs`` folder.
+
+
+Types
+^^^^^
+
+There are three kinds of GDPs:
+
+1. A **Feature** GDP describes a new feature or implementation
+   for GT4Py or GTScript.
+
+2. An **Informational** GDP describes a GT4Py design issue, or provides
+   general guidelines or information to the community, but does not propose
+   a new feature. Informational GDPs do not necessarily represent a GT4Py
+   community consensus or recommendation.
+
+3. A **Process** GDP describes a process surrounding GT4Py, or proposes a
+   change to a process. Process GDPs are like Feature GDPs but apply to
+   areas other than the GT4Py framework itself. Examples include
+   procedures, guidelines, changes to the decision-making process, and
+   changes to the tools or environment used in GT4Py development.
+   Any meta-GDP is also considered a Process GDP.
+
+
+GDP Workflow
+------------
+
+The GDP process begins with a new idea for GT4Py. It is highly
+recommended that a single GDP contains a single key proposal or new
+idea. Small enhancements or patches often don't need a GDP and can be
+injected into the GT4Py development workflow with a pull request to
+the GT4Py `repo`_. The more focused the GDP, the more successful it tends
+to be. If in doubt, split a large GDP into several well-focused ones.
+
+Each GDP must have a champion—someone who writes the GDP using the style
+and format described below, shepherds the discussions in the appropriate
+forums, and attempts to build community consensus around the idea. The GDP
+champion (a.k.a. Author) should first attempt to ascertain whether the idea is
+suitable for a GDP. Posting to the GT4Py `slack`_  channel is the best
+way to start.
+
+The proposal should be submitted as a draft GDP via a `GitHub pull
+request`_ to the ``doc/wiki/GDPs`` directory with the name ``gdp-<n>.rst``
+where ``<n>`` is an appropriately assigned four-digit number (e.g.,
+``gdp-0000.rst``). The draft must use the :doc:`gdp-template` file.
```

Comment:
template link broken

Reply:
I think this only works properly in the rendered output, not in GitHub preview. I would leave it as it is unless it doesn't work properly when we clone the GDPs folder to the project wiki

Reply:
no, it points to GDP 0 instead of the template

Reply:
It's actually the same issue as before: only works properly in the HTML rendered by Sphinx not in Github, where it seems to link to this very same page because I guess github it does not know how to deal with the markup. I can test if the same problem happens in the wiki pages, and  then try to find a solution, or we can leave as it is for now, since this is the right way to express it for sphinx, which in the end is the doc generator.